### ISSUE TITLE
ts.updategeochronanalunit issue

### ISSUE THREAD

**Issue by ecgrimm** [NeotomaDB/tilia_api#9]
ts.updategeochronanalunit isn't working. Get error: "HTTP protocol error. 500 Internal Server Error."

Tried these input params: geochronid = 39437, analysisunitid = 338143, depth = 200, thickness = 1.

**Comment by SimonGoring** [comment]
@ecgrimm I just tried to adjust this, but right now in `neotomatilia` there is no collection unit that contains the analysis unit 338143.  This may be why the function doesn't work on `tilia-dev`.  I'm going to try to pull another `geochronid` that will work.

The other issue is that right now the function seems to be called `ts.updategeochronanalysisunit` as opposed to the name you gave it.

**Comment by ecgrimm** [comment]
@SimonGoring Right. Analysis unit 338143 is a new upload to the production database.

**Comment by ecgrimm** [comment]
@SimonGoring It is `ts.updategeochronanalysisunit`, and that's the name I'm using. Darn, I was hoping that was the problem!
